drivers/flash/flash_mspi_atxp032.c

```diff
@@ -31,7 +31,7 @@ LOG_MODULE_REGISTER(flash_mspi_atxp032, CONFIG_FLASH_LOG_LEVEL);
 #define NOR_WRITE_SIZE                      1
 #define NOR_ERASE_VALUE                     0xff
 
-#define ATXP032_VENDOR_ID                   0x43
+#define ATXP032_VENDOR_ID                   0x1F
```

Comment:
Something is not right here, because adesto vendor number is 0x43, but their spec shows 9 bits in ID
![image](https://github.com/user-attachments/assets/48ed69fb-a9f9-4782-b945-4ca2f5d6aea3)

Reply:
ah, the one on my board is atxp128. Its datasheet says 0x1F as manufacturer ID LOL.
<img width="581" alt="image" src="https://github.com/user-attachments/assets/2c2ea7a4-1193-4cf3-87cf-3219faacce91">
I will try to find a atxp032 and verified the ID again.

Reply:
The vendor ID is indeed 0x43 at the 7th position.
![image](https://github.com/user-attachments/assets/26592afa-5990-49c5-8a05-6a6f50435781)

Reply:
And one more comment, the 0xfe in the front, which  appears seven times, is actually part of id.
We can not fix that now, so we have to stick with 3 byte id.

Reply:
> And one more comment, the 0xfe in the front, which appears seven times, is actually part of id. We can not fix that now, so we have to stick with 3 byte id.

yeah, but 0xfe is not what jedc wants...
Probably going to file another PR that adds support for ATXP064 and ATXP128 which ID 0x1F appears on the first byte instead of the 8th...